modules/sr/robot3/marker.py

```diff
@@ -216,18 +216,23 @@ class Orientation:
         angle=pi, axis=(1, 0, 0),
     )
 
-    def __init__(self, rotation_matrix: tuple[float, float, float, float]):
+    @classmethod
+    def from_axis_angle(cls, axis_angle: tuple[float, float, float, float]) -> Orientation:
+        x, y, z, rad = axis_angle
+
+        # Calculate the quaternion of the rotation in the camera's coordinate system
+        # Correct for roll and yaw turning the wrong way
+        quaternion = Quaternion.from_angle_axis(angle=rad, axis=(-x, y, -z))
```

Comment:
Can we not move the axis conversion from the init as we want to keep the quaternion being passed in webots' coordinate system.